src/router/Router.ts

```diff
@@ -2,6 +2,7 @@ import { createVNode } from 'inferno';
 import Component from 'inferno-component';
 import match, { matchPath } from './match';
 import RouterContext from './RouterContext';
+import VNodeFlags from 'inferno-vnode-flags';
```

Comment:
bc `VNodeFlags` is a package now and no long a global `enum` that gets injected. This makes the dependency explicit instead of tracing down global dep :)